### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by lic-8** [G4lile0/Heimdall-WiFi-Radar#3]
Hi there !
Just found your amazing work here, but I can't find any kind of documentation or instructions for basic usage. Am I missing something ?
Thanks !

**Comment by BuckerooBanza1** [comment]
Yeah, i think we would love to see how to setup and use this Tool.
